src/libraries/System.ComponentModel.TypeConverter/ref/System.ComponentModel.TypeConverter.cs

```diff
@@ -686,6 +686,7 @@ public partial interface ITypeDescriptorContext : System.IServiceProvider
     }
     public partial interface ITypedList
     {
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Members of property types might be trimmed if not referenced directly")]
```

Comment:
Is this `RequiresUnreferencedCodeAttribute` still necessary after all the other removals we've made? I'm looking through the 2 implementations of this method, but it is hard to tell.

Reply:
good catch, just checked - not needed